code/addons/a11y/src/types.ts

```diff
@@ -1,3 +1,47 @@
 import type { AxeResults } from 'axe-core';
 
 export type A11YReport = AxeResults | { error: Error };
+
+export interface A11yParameters {
+  /**
+   * Accessibility configuration
+   *
+   * @see https://storybook.js.org/docs/writing-tests/accessibility-testing
+   */
+  a11y?: {
+    /** Manual configuration for specific elements */
+    element?: string | string[];
+
+    /** Configuration for the accessibility rules */
+    config?: {
+      /** Rules to run against the matching elements */
+      rules?: Array<{
+        id: string;
+        enabled?: boolean;
+        selector?: string;
+      }>;
+      /** Elements to exclude from accessibility checks */
+      exclude?: string[];
+    };
+
+    /**
+     * Options for the accessibility checks To learn more about the available options,
+     *
+     * @see https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter
+     */
+    options?: Record<string, any>;
+
+    /** Turn off this addon's behavior */
```

Comment:
```suggestion
    /** Remove the addon panel and disable the addon's behavior */
```

Plus other instances below?